support/package_all

```diff
@@ -2,44 +2,47 @@
 #
 # Package all extensions: both internal and non-internal.
 
-if [ -n "$DEBUG" ]; then
-    set -x
+if [ -n "${DEBUG}" ]; then
+	set -x
 fi
 
-if [ -z "$1" ]; then
-    echo "Usage: $0 PHP_VERSION"
-    echo "Example: $0 7.4"
-    exit 1
+if [ -z "${1}" ]; then
+	echo "Usage: ${0} PHP_VERSION"
+	echo "Example: ${0} 7.4"
+	exit 1
 fi
 
 set -e
 set -o pipefail
 
-php_version=$1
+php_version="${1}"
 
-cd $( cd -P "$( dirname "$0" )" && pwd )
+cd $( cd -P "$( dirname "${0}" )" && pwd )
 
 for e in ./ext-internal/*; do
-  ext_name="$( basename "${e}" )"
-  # IMAP extension has been moved to PECL starting with PHP 8.4
-  # https://www.php.net/ChangeLog-8.php#8.4.1
-  if [[ "$ext_name" = "imap" ]] && [[ "$php_version" = "8.4" ]]; then
-    continue
-  fi
-
-  ./package_internal_ext "${ext_name}" "${php_version}"
-  if [[ $? -ne 0 ]]; then
-    echo "Error packaging internal extension ${ext_name}" >&2
-    exit 1
-  fi
+	ext_name="$( basename "${e}" )"
+	# IMAP extension has been moved to PECL starting with PHP 8.4
+	# https://www.php.net/ChangeLog-8.php#8.4.1
+	if [[ "${ext_name}" == "imap" ]] \
+		&& [[ "${php_version}" == "8.4" || "${php_version}" == "8.5" ]]
```

Comment:
suggestion: would it be possible to write something like:
```suggestion
		&& [[ "${PHP_MODULE_API_VERSIONS["$php_series"]}" -ge 20240924 ]]
```

That way we wouldn't forget to update this for the next major release